Altis_Life.Altis/core/functions/fn_clearVehicleAmmo.sqf

```diff
@@ -9,26 +9,31 @@
     Syntax: _vehicle removeMagazinesTurret [magazineName, turretPath]
     Documentation: https://community.bistudio.com/wiki/removeMagazinesTurret
 */
-private ["_vehicle","_veh"];
-_vehicle = [_this,0,objNull,[objNull]] call BIS_fnc_param;
+
+params [["_vehicle", objNull, [objNull]];
```

Comment:
Your params is broken
`params [["_vehicle", objNull, [objNull]];`
should be
`params [
["_vehicle", objNull, [objNull]]
];`